akka-persistence-tck/src/main/scala/akka/persistence/snapshot/SnapshotStoreSpec.scala

```diff
@@ -52,6 +52,8 @@ abstract class SnapshotStoreSpec(config: Config) extends PluginSpec(config)
     }
   }
 
+  def snapshotByteSizeLimit = 10000
```

Comment:
Maybe add a scaladoc on this mentioning that it can be overridden to a limit suitable for the specific implementation but that authors should also document that?